rg_instructor_analytics/tests/test_view.py

```diff
@@ -0,0 +1,149 @@
+"""Test for view."""
+from datetime import date
+
+from django.http.request import QueryDict
+from django.test import RequestFactory, TestCase
+from mock import Mock, patch
+from opaque_keys import InvalidKeyError
+
+from rg_instructor_analytics.views import EnrollmentStatisticView
+
+
+class TestEnrollmentStatisticView(TestCase):
+    """Test for enrollment statistic view."""
```

Comment:
format class description in OpenEdx style please

Reply:
done